zebrad/src/components/tracing/component.rs

```diff
@@ -266,7 +270,23 @@ impl Tracing {
             info!("activated progress bar");
 
             if config.log_file.is_some() {
-                eprintln!("waiting for initial progress reports...");
+                // Clear screen
+                eprint!("\x1B[2J");
```

Comment:
These escape codes should only be sent if output is going to a terminal: `ZebradApp::outputs_are_ttys()`

Reply:
The `use_color` variable in this method is `true` if you can use terminal colour escapes.

It's refactored in PR #6997, but the behaviour of tracing doesn't change in that PR.

Reply:
I did that on da446250, with the difference that I test stderr, because the logo is printed there (and that's because the progress bars are also printed there. Which makes sense overall)